lib/stuart_client_elixir/http_client.ex

```diff
@@ -71,7 +71,12 @@ defmodule StuartClientElixir.HttpClient do
   end
 
   defp to_api_response({:ok, %HTTPoison.Response{status_code: status_code, body: body}}) do
-    %{status_code: status_code, body: Jason.decode!(body)}
+    with {:ok, decoded_body} <- Jason.decode(body) do
+      %{status_code: status_code, body: decoded_body}
+    else
```

Comment:
I think we can get rid of this `else`, because the default behavior of a `with` clause is returning what doesn't match. Unless we're doing a special treatment of the error, logging it, etc., we don't need the `else`.